lib/tests/streamlit/string_util_test.py

```diff
@@ -13,15 +13,32 @@
 # limitations under the License.
 
 import unittest
+from parameterized import parameterized
 
 from streamlit import string_util
 
 
 class StringUtilTest(unittest.TestCase):
     def test_decode_ascii(self):
-        """Test streamlit.util.decode_ascii."""
+        """Test streamlit.string_.decode_ascii."""
```

Comment:
nit: should be `streamlit.string_util.decode_ascii`

Reply:
Fixed it (also for all other docstrings in this file)